content/post/leadership-ci.md

```diff
@@ -0,0 +1,133 @@
++++
+date = "2019-04-06T8:09:26-07:00"
+title = "Leadership CI"
+author = "Jessica Frazelle"
+description = "Questions to ask yourself to make sure you are in the right mindset to think and behave as a leader."
++++
+
+This post is co-authored by [Kathy Simpson](https://github.com/simpsoka).
+
+> “understanding the true nature of instinctive decision making requires us to be forgiving of those people trapped in circumstances where good judgment is imperiled.” 
+> ― Malcolm Gladwell, Blink: The Power of Thinking Without Thinking
+
+
+As leaders, setting up a structure that helps us navigate decisions under
+pressure is of the utmost importance. 
+When writing and delivering software we rely on our
+continuous integration (CI) infrastructure and test suites to tell us when a test is
+failing and code should not be merged.
+
+As leaders, before acting or making decisions it would be nice to have a set of
+tests and checks, established ahead of time, to make sure we are in the
+right headspace to think, behave and make decisions that are in the best 
+interest of everyone and our company. There are devastating consequences to
```

Comment:
Added this sentence, hope it makes sense :) "There are devastating consequences to taking actions based on fear and pride; we hope this set of questions enables taking action based on growth, humility, inclusion, and soulful reflection."